fastcrypto-cli/src/vdf.rs

```diff
@@ -133,14 +116,14 @@ fn execute(cmd: Command) -> Result<String, Error> {
                     "Invalid input point or discriminant.",
                 )
             })?;
-            if !g.has_parameter(&discriminant) {
+            if !g.has_parameter(&DISCRIMINANT_3072) {
                 return Err(Error::new(
                     ErrorKind::InvalidInput,
                     "Input point does not match discriminant.",
                 ));
             }
 
-            let vdf = DefaultVDF::new(discriminant, arguments.iterations);
+            let vdf = DefaultVDF::new(DISCRIMINANT_3072.clone(), arguments.iterations);
```

Comment:
is there a reason this is not a reference?

Reply:
It's because the VDF must store the discriminant for future use, so it'll be cloned in the constructor anyway.